samples/jigsaw_puzzle/jigsaw-prototype.py

```diff
@@ -1,6 +1,6 @@
 import cv2
```

Comment:
Note: to run this, need to install extra lib `pip install opencv-python`

Reply:
Added a comment in the python file